Formula/gomplate.rb

```diff
@@ -16,10 +16,6 @@ class Gomplate < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "621a2a033543fc50ce3ad023d16b1d36949e6acc611ecfe9aeef3c75f9c04217"
   end
 
-  # upstream issue report, https://github.com/hairyhenderson/gomplate/issues/1614
-  # update to use go@1.20 when the issue is resolved
-  depends_on "go@1.19" => :build
```

Comment:
To use the latest `go`, you need to change this to

```ruby
  depends_on "go" => :build
```

Reply:
➕